CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [4.2.1](https://github.com/algolia/angular-instantsearch/compare/4.2.0...4.2.1) (2022-06-13)
+
+
+### Bug Fixes
```

Comment:
should the angular peer dependency be a fix here too? we marked it as chore, but probably was a fix